mergify_engine/context.py

```diff
@@ -822,9 +823,33 @@ async def _get_consolidated_data(self, name):
                 for ctext, state in (await self.checks).items()
                 if state == "stale"
             ]
+        elif name == "depends-on":
+            # TODO(sileht): This is the list of merged pull request that
+            # required-by this pull request
```

Comment:
That sounds wrong no? That's the list of pull request that this PR requires, right?

Reply:
The auto generates rule conditions have all `Depends-On:` but the consolidated_data only have the merged one.

This way, the condition will match only when the pull request is merged.

Reply:
Ok, we're saying the same thing, it's just the comment format make me thought you meant the other way around.
```suggestion
            # This is the list of merged pull requests that are required by this pull request
```